### ISSUE TITLE
Build failure using gradle to build java project

### ISSUE THREAD

**Issue by ignasi35** [akka/akka-grpc#1034]
Preparing `akka-grpc-quickstart-java` for the `1.0.0` release the following issue came up:

```
* What went wrong:
An exception occurred applying plugin request [id: 'com.lightbend.akka.grpc.gradle', version: '1.0.0']
> Failed to apply plugin [id 'com.lightbend.akka.grpc.gradle']
   > Could not find method protobuf() for arguments [akka.grpc.gradle.AkkaGrpcPlugin$_apply_closure1$_closure2@2e1a0b02] on root project 'akka-grpc-quickstart-java' of type org.gradle.api.Project.
```

See https://github.com/akka/akka-grpc-quickstart-java.g8/pull/38.

The build in https://github.com/akka/akka-grpc-quickstart-java.g8/pull/38 is a 1:1 port of https://github.com/akka/akka-grpc-quickstart-scala.g8/pull/43. In the scala sampel project, the import of `com.google.protobuf` and the usage of its `protobuf` command works as expected.

### Steps to reproduce:

```
sbt new akka/akka-grpc-quickstart-java.g8
## Accept all defaults
cd akka-grpc-quickstart-java
./gradlew test
```

Compare that to:

```
sbt new akka/akka-grpc-quickstart-scala.g8
## Accept all defaults
cd akka-grpc-quickstart-scala
./gradlew test
```

**Comment by raboof** [comment]
Very curious that it works for the scala quickstart and for both akka-grpc gradle travis jobs, but not for the java quickstart.

The error rings a bell but I can't place it...

**Comment by eshepelyuk** [comment]
Should be fixed by #1054

**Comment by raboof** [comment]
> Should be fixed by #1054

Confirmed it does, tested with akka-grpc snapshot version 1.0.0-24-1ff30e97 (adding the snapshot repository to the gradle build as `maven { url 'http://dl.bintray.com/akka/snapshots' }`)
